llvm/lib/ProfileData/Coverage/CoverageMapping.cpp

```diff
@@ -580,6 +583,72 @@ static unsigned getMaxBitmapSize(const CounterMappingContext &Ctx,
   return MaxBitmapID + (SizeInBits / CHAR_BIT);
 }
 
+struct DecisionRow {
+  const CounterMappingRegion *DecisionRegion;
+  LineColPair DecisionStartLoc;
+  LineColPair DecisionEndLoc;
+
+  SmallVector<const CounterMappingRegion *, 6> Branches;
```

Comment:
why 6?

Reply:
For now, the maximum number of conditions is 6.

Reply:
Worth a comment?